index.html

```diff
@@ -219,7 +280,12 @@ <h5 class="text-white">Skills</h5>
         <h3 class="form-section-content-h3">
           I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </h3>
-      </section> 
+      </section>
+      
+
+      <div class="form-and-footer-section">
+
+        
       <form action="https://formspree.io/f/xnqwzypz" method="post">
         <ul>
           <li class="form-li">
```

Comment:
- [ ] For the form in the desktop version Please fix the width of the form elements and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159835816-c76dddae-ed07-4687-8526-c119cc8820d9.png)  | ![image](https://user-images.githubusercontent.com/68796232/159835764-41f0bae4-588a-4db6-9d19-560c0f1d18ff.png)  |

Reply:
### I fixed this but as a result of what Microverse said that we should not mind the other full name, that is the reason I did not give it the width.
![cssnew](https://user-images.githubusercontent.com/86200356/159942361-930db518-3801-4176-8fcb-f35edd3bda83.jpg)